atd-vze/src/views/Tools/CreateCrashRecordTable.js

```diff
@@ -138,38 +144,36 @@ const CreateCrashRecordTable = () => {
                 <tr>
                   <th>Crash ID</th>
                   <th>Case ID</th>
-                  <th>Crash Date</th>
-                  <th>Crash Time</th>
+                  <th>Crash Timestamp</th>
                   <th>Updated By</th>
-                  <th>Last Update</th>
+                  <th>Updated At</th>
                   <th>Action</th>
                 </tr>
               </thead>
               <tbody>
                 {crashesData &&
-                  crashesData["atd_txdot_crashes"] &&
-                  crashesData.atd_txdot_crashes.map((item, index) => {
+                  crashesData["crashes"] &&
+                  crashesData.crashes.map((item, index) => {
                     if (crashSearch !== "" && item.case_id !== crashSearch)
                       return null;
 
                     return (
                       <tr key={index}>
                         <td>
-                          <Link to={`/crashes/${item.crash_id}`}>
-                            {item.crash_id}
+                          <Link to={`/crashes/${item.record_locator}`}>
+                            {item.record_locator}
                           </Link>
                         </td>
                         <td>{item.case_id}</td>
-                        <td>{item.crash_date}</td>
-                        <td>{item.crash_time}</td>
+                        <td>{item.crash_timestamp}</td>
                         <td>{item.updated_by}</td>
-                        <td>{item.last_update}</td>
+                        <td>{item.updated_at}</td>
                         <td>
                           <Button
                             color="danger"
                             className="btn-pill"
                             size={"sm"}
-                            onClick={() => openModalDelete(item.crash_id)}
+                            onClick={() => openModalDelete(item.record_locator)}
```

Comment:
I tracked down the error im getting when I try to delete here, its sending the record locator "T32449" to the delete mutation on line 36 that is expecting an Int!